src/fiatExchanges/SelectProvider.tsx

```diff
@@ -57,6 +61,36 @@ const TAG = 'SelectProviderScreen'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.SelectProvider>
 
+export function getAddFundsCryptoExchangeExperimentParams() {
+  const { addFundsExchangesLink, addFundsExchangesText } =
```

Comment:
nit: IMO the destructuring here makes the code more confusing because its not clear what `addFundsExchangesLink` and `addFundsExchangesLink` are. It might be clearer, if a little more verbose, to just call this variable `params` and reference that variable through the rest of the function 
```js
const exchangesText = experiment.get(
      params.addFundsExchangesText.paramName,
      params.addFundsExchangesText.defaultValue
    )
```

don't feel too strongly.

Reply:
updated, I was also not too convinced with the destructuring